packages/companies/companies.txt

```diff
@@ -368,6 +368,7 @@ Genworth Financial, Inc.
 George Weston
 GeoTrust Global
 Gilead Sciences, Inc.
+Gitlab
```

Comment:
Misspelling, the official version of the name is: `GitLab`
(with big `L` character).